Program.cs

```diff
@@ -107,13 +129,12 @@ static void DisplayHangman(int incorrectGuesses)
             ConsoleColor.Magenta // 6 Right leg
         };
 
-
         Console.WriteLine("Hangman: Toyota Edition!");
-        Console.WriteLine("  ______");
-        Console.WriteLine(" /|_||_\\`.___");
-        Console.WriteLine("(   _TOYOTA_ \\");
-        Console.WriteLine("=`-(_)----(_)-'");
-
+        Console.WriteLine(@"
```

Comment:
Much better code readability to only have "Console. WriteLine" written once for the ASCII Art. Only thing I'm wondering is what do you need the "@" symbol for on the car art?